sdk/azcore/cloud/cloud.go

```diff
@@ -0,0 +1,44 @@
+//go:build go1.16
+// +build go1.16
```

Comment:
The build constraints will need to be updated to go1.18 after rebasing on `main`.